app/src/components/map/wfs-utils.tsx

```diff
@@ -39,9 +45,9 @@ export const layerContentHandlers: Record<string, any> = {
             key={`${feature.id}-game-management-zone-id`}>{`Game Management Zone: ${feature.properties.GAME_MANAGEMENT_ZONE_ID}`}</div>
           <div
             key={`${feature.id}-game-management-zone-name`}>{`Game Management Zone Name: ${feature.properties.GAME_MANAGEMENT_ZONE_NAME}`}</div>
-          <div key={`${feature.id}-area`}>{`Region Area: ${(feature.properties.FEATURE_AREA_SQM / 10000).toFixed(
-            0
-          )} ha`}</div>
+          <div key={`${feature.id}-area`}>
+            {`Region Area: ${(feature.properties.FEATURE_AREA_SQM / 10000).toFixed(0)} ha`}
```

Comment:
You may want to chain `.toLocaleString()`, which formats a number like `1234567` into `1,234,567`

Reply:
good suggestion, added.